data/tutorials/lg_02_if_statements_loops_and_recursion.md

```diff
@@ -823,22 +828,22 @@ let rec loop () =
   | recursive case -> element :: loop ()
 ```
 The key here is actually the use of the match / base case / recursive
-case pattern. In this example - finding the maximum element in a list -
-we're going to have two base cases and one recursive case. But before I
+case pattern. In this example (finding the maximum element in a list),
+we'll have two base cases and one recursive case. But before we
 jump ahead to the code, let's just step back and think about the
-problem. By thinking about the problem, the solution will appear "as if
-by magic" (I promise you :-)
+problem. By thinking about the problem, the solution will appear *as if
+by magic*. (I promise you :-)
```

Comment:
We might as well remove the "I promise you" (I believe _something_ is expected after the "you", and something like "I swear" is more correct)

Reply:
Nice insight!